queries/java/highlights.scm

```diff
@@ -133,145 +132,150 @@
 
 ; Literals
 
-[
-(hex_integer_literal)
-(decimal_integer_literal)
-(octal_integer_literal)
-(binary_integer_literal)
-] @number
+(string_literal) @string
```

Comment:
`text_block` is now also just a `string_literal` ?

Reply:
yes, they're separated by hidden rules, iirc _string_literal and _multiline_string_literal